src/components/data/data-utils.js

```diff
@@ -928,6 +928,104 @@ export const MultipleEntityEditor = ({ entityType, entityNames, attributeNames,
   ])
 }
 
+export const AddColumnModal = ({ entityType, entityMetadata, workspaceId: { namespace, name }, onDismiss, onSuccess }) => {
+  const [columnName, setColumnName] = useState('')
+  const [columnNameTouched, setColumnNameTouched] = useState(false)
```

Comment:
Nice! This is definitely one of those subtle UX things and I love how clearly this expresses the intent of the code.